utils/cryptodev-linux/Makefile

```diff
@@ -10,12 +10,15 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=cryptodev-linux
-PKG_VERSION:=1.8
-PKG_RELEASE:=2
+PKG_VERSION:=2017-02-09
```

Comment:
Think we can switch to the related git version of the 1.9 version when it will be released...
Anyway 1.8 is very old (2015), think it's better to update...